.github/workflows/scripts/comments.mjs

```diff
@@ -40,13 +93,49 @@ export async function comment ({ github, context }, issueNumber, markerText, bod
   /**
    * Create GitHub issue comment
    *
-   * Updates existing commentby ID if available
+   * Updates existing comment by ID if available
    */
   await (!commentId
     ? github.rest.issues.createComment({ ...issueParameters, body })
     : github.rest.issues.updateComment({ ...issueParameters, body, comment_id: commentId }))
 }
 
+/**
+ * @param {Pick<GithubActionContext, "context" | "commit">} githubActionContext
+ * @returns {string} - The content for the footer
+ */
+function renderCommentFooter ({ context, commit }) {
+  return `Commit: ${commit} | [Action run](${githubActionRunUrl(context.runId)})`
```

Comment:
I've strayed away from dates. While we could pick a timezone and use the `Intl` API for formatting, it wouldn't be clear to anyone reading the comment. Plus the info is kind of available under the "edited" dropdown.

Reply:
Where workflow runs have been run again, are we alright linking to the latest?

The comment might have been written by a previous run:

https://github.com/alphagov/govuk-frontend/actions/runs/4894152384/attempts/1
https://github.com/alphagov/govuk-frontend/actions/runs/4894152384/attempts/2
https://github.com/alphagov/govuk-frontend/actions/runs/4894152384

Haven't thought too much about GitHub comments + screen readers but is `|` "Vertical Bar" alright?

```console
Commit: ${commit}
Workflow run: ${githubActionRunUrl(context.runId)}
```

Reply:
Good shout on the accessibility, I'll update, maybe to something smaller like "[Workflow run](url-of-fun) for [CommitSHA](url-of-commit)" so we eat as little characters as possible for that footer.

Good spot also for the attempts, I'll see if I can dig the URL to the specific attempt (hopefully it's somewhere in the `context` object or something else). If I cannot, we'll have to set it to the latest run.